server/setup.py

```diff
@@ -28,5 +28,16 @@
     install_requires=[
         'blinker', 'celery >=3.1.0, <3.2.0', DJANGO_REQUIRES, 'httplib2', 'iniparse',
         'isodate>=0.5.0', 'm2crypto', 'mongoengine>=0.10.0', 'oauth2>=1.5.211', 'pymongo>=3.0.0',
-        'semantic_version>=2.2.0', 'setuptools']
+        'semantic_version>=2.2.0', 'setuptools'],
+    data_files=[
```

Comment:
It did work when I ran `setup.py install`, but wasn't working when I tried `setup.py develop`. If there's a better way to do this, I'd definitely like to know.

My main goal was to not force the spec file to use `install`